tests/binary-cache-build-remote.sh

```diff
@@ -1,5 +1,8 @@
 source common.sh
 
+enableFeatures nix-command
+unknownFailsOnNixOS # fails because of path may have been built and not gc-ed; generate unique derivation?
```

Comment:
We'll want to resolve some of these later, so that we don't have to block this PR for too long.